content/competitions/codejam.html

```diff
@@ -8,19 +8,15 @@
     <section class="column is-6">
 
         <div class="block">
-            <a href="https://www.facebook.com/events/659245534649313/">
-                <img src="{{< baseurl >}}/img/media/codejam_banner.png" style="width: 100%" />
+            <a href="https://fb.me/e/42v91HFQA">
+                <img src="/img/media/codejam_banner.png" style="width: 100%" />
             </a>
         </div>
 
-        <h1 class="title is-3">UQCS CodeJam 2020 (presented by Atlassian)</h1>
-        <h2 class="subtitle is-4">10am &ndash; 3:30pm, Saturday 28th March</h2>
+        <h1 class="title is-3">UQCS CodeJam 2021 (Presented by IMC)</h1>
+        <h2 class="subtitle is-4">10am &ndash; 4pm, Saturday 20th March</h2>
 
         <div class="content">
-            <h4>
-                <a href="https://www.hackerrank.com/uqcs-codejam-2020">Contest now open on HackerRank</a>.<br />
-                Sign up to compete and get on the leaderboard!
-            </h4>
 
             <p>The UQCS CodeJam is an individual competitive programming competition. You get a list of questions to
```

Comment:
This is a pretty lame opening sentence. Probably not your work, but since you're working on it ;)

Reply:
This would've been my work 😔 It can definitely be better.